servicepulse/intro-failed-message-retries.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Retrying failed messages
+summary: Describes the concept and lifecycle of retrying messages in ServicePulse
+component: ServicePulse
+tags:
+- ServicePulse
+reviewed: 2016-12-15
+---
+
+After addressing the root cause of the message's processing failure, failed messages can be resent for reprocessing by the corresponding endpoint(s). This is referred to as a "retry" (or a manual retry, in contrast to the automated and configurable [Recoverability](/nservicebus/recoverability/)).
```

Comment:
> After addressing the root cause of the message's processing failure

That's really complicated, why not just "After resolving the issue that caused processing failures,... "